recipes/scispacy/post-link.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python3 -m pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.5.4/en_ner_bc5cdr_md-0.5.4.tar.gz --no-deps --no-build-isolation --no-cache-dir --use-pep517 -vvv
```

Comment:
Isn't this the same as https://github.com/bioconda/bioconda-recipes/blob/ff0b6ac20c263830bd0a1d00424488a1c37b1de2/recipes/scispacy-en_ner_bc5cdr_md/meta.yaml#L18